### DISCUSSION TITLE
Where's the reference docs?

### DISCUSSION THREAD

**Discussion by teknolog2000** [Shopify/polaris-react#6609] [General] [answered]
A few months ago I built an internal app using Polaris. I relied extensively on reference documentation, which since seems to have been purged from the internet. I'm pretty sure they were on shopify.dev, but now they're not.

So where are they? How am I supposed to use Polaris without this? Right now I wanted to see the properties for a TextField, but the only page I find this [this one](https://polaris.shopify.com/components/text-field), which is pretty much useless.

**Comment by jared-fanning** [comment, marked as answer]
Hi @teknolog2000, props are visible on the website again! :slightly_smiling_face: We will be shipping more improvements to this feature in the coming weeks.

For now, you can also check out the previous version of the doc site here: 
https://legacy.polaris.shopify.com/components/forms/text-field#navigation
